source/index.rst

```diff
@@ -12,8 +12,9 @@ Welcome to the OSL Wiki!
     gsoc/index
     ganeti/index
     openstack/index
-    powerci/index
-    ibmz-ci/index
+    jenkins/powerci
+    jenkins/ibmz-ci
+    jenkins/jenkins-docker
```

Comment:
Instead of doing it this way, let's create an index under the `jenkins` folder (see howtos) and then reference that here. We may want to also consider showing more than one level of index here, but I can't remember if you can do that here or not.

Reply:
I edited the table of contents to list all the pages in the wiki; is it clearer that way or does it look too cluttered now do you think?

Reply:
@ArtofBugs it looks good to me, my only suggestion is to remove the `gsoc/index` from the TOC as we haven't been a part of that for many years. People can still get to it on the side bar however.

Reply:
Sounds good!